### ISSUE TITLE
Some stations that exists in the deparute board can not be used for manual checkins

### ISSUE THREAD

**Issue by f2k1de** [derf/travelynx#100]
When I want to check into a train calling at (for example) Huddersfield, this is possible via the departure board.

<img width="300" alt="Bildschirmfoto 2023-11-27 um 09 23 24" src="https://github.com/derf/travelynx/assets/11199213/a2a423fe-e781-4be6-9e93-18edb7ea203c">

But if I've missed the checkin because of time zone offset (or other reasons) and want to add it manually, travelynx is not aware of this station and I cant add it to my route. 

<img width="300" alt="Bildschirmfoto 2023-11-27 um 09 23 15" src="https://github.com/derf/travelynx/assets/11199213/7eb426be-20a1-4b97-a86f-26bff2da249a">

**Comment by derf** [comment]
Yup, HAFAS support in manual journey entry mode is still on the to-do list.

**Comment by Ein-Tim** [comment]
Resolved with db-RIS integration?
